.github/workflows/lint-pr.yml

```diff
@@ -12,16 +12,12 @@ jobs:
     name: Validate PR title
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v5
-        with:
-          types: |
-            fix
-            feat
-            test
-            refactor
-            chore
-          requireScope: false
-          scopes: |
-            scope-not-allowed
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          if [[ "${{ github.event.pull_request.title }}" =~ (^(chore|feat|fix|refactor|test){1}?: ([[:alnum:]])+([[:space:][:print:]]*)) ]]; then
```

Comment:
The sign off doc refers to conventional commits where an optional scope is part of. This regex does not check for an optional scope in the title. Is this intended?

Reply:
All clear now. LGTM!